include/SFML/Window/ContextSettings.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef SFML_CONTEXTSETTINGS_HPP
 #define SFML_CONTEXTSETTINGS_HPP
 
+#include <SFML/Config.hpp>
```

Comment:
That might well be, yes! I'll mark this as a requested change and deal with it later.